### ISSUE TITLE
Default profile message category

### ISSUE THREAD

**Issue by buddh4** [humhub/humhub#4185]
Currently the default message category of profile fields is `UserModule.profile`, but those translation files are possibly overwritten by core updates. We should define an empty root category for custom user translations, which won't be overwritten by any updates.

**Comment by ArchBlood** [comment]
You should be able to use `/protected/config/messages` for translating no?

### Example
- Create `UserModule.demo.php` in `/protected/config/messages/ja`
- Create new field/category place `UserModule.demo` in Translation ID field and save.

To be honest users should be encouraged to use Translation overwrite for such a thing regardless, you don't even have to add a full `UserModule.profile.php` you can use this method to only translate parts that are newly created or overwrite text that's already there. The main issue would be translations that have no Module Translation ID (e.g `/protected/humhub/messages`), this should have it's own unique translation ID because those files can not be overwritten by default.

**Comment by buddh4** [comment]
You are right in the case of profile fields,  you could/should use the message overwrite/configuration mechanism. As fallback for cases without direct module relation  you should be able to use the 'base' category.

**Comment by ArchBlood** [comment]
@GreenMeteor has tried the base case usage and it has not worked sadly. I'm only guessing this because when we use `{ModuleID}.{MessageFile}.php` as suggested in the [docs](https://docs.humhub.org/docs/admin/translations#overwrite-translation-messages) translations work flawlessly, but in cases where no module ID is it fails see [here](https://community.humhub.com/s/theming-appearance/?contentId=219707) on the conversation that's already been done about this topic. :wink:

**Comment by buddh4** [comment]
Can we add this behavior for non module message files @luke- ?

**Comment by funkycram** [comment]
+1

**Comment by luke-** [comment]
@funkycram Good point. We should then possibly also document standard texts such as (Back, Save, To overview, ...) and try to use them centrally there.

**Comment by samuk** [comment]
I'm keen to get this feature so that we can customise error pages and ask users non-logged-in
![login](https://user-images.githubusercontent.com/400875/206682091-9ab796cb-74c4-40f9-85cd-2c11c4181183.png)
 users to login rather than return to a dashboard which will not fix their problem.
